lib/CleantalkSP/SpbctWP/Firewall/UploadChecker.php

```diff
@@ -301,6 +306,26 @@ public static function runCheckForWordpressModules(
         // Output result message
         if ( $overall_result ) {
             if (self::checkVulnerability($source)) {
+                $name_file = $_FILES['pluginzip']['name'];
+
+                ?>
+                <b>If you want to continue the installation, add the file and click install.</b>
+                <form method="post" enctype="multipart/form-data" class="wp-upload-form" 
+                action="<?php echo esc_url(self_admin_url('update.php?action=upload-plugin')); ?>">
+                    <?php wp_nonce_field('plugin-upload'); ?>
+                    <label class="screen-reader-text" for="pluginzip">
+                        <?php
+                        /* translators: Hidden accessibility text. */
+                        _e('Plugin zip file');
+                        ?>
+                    </label>
+                    <input type="hidden" name="trust_module" value="true">
+                    <input type="hidden" name="file_path" id="file_path" value="<?php echo (wp_get_upload_dir()['path'] . '/' . $name_file) ?>">
```

Comment:
The value mast to be sanitized by `escattr()`

Reply:
the output in value must also be deleted